src/routes/index.js

```diff
@@ -15,3 +15,10 @@ router.post('/fetch-ka', (req, res) => {
 });
 
 module.exports = router;
+
+const client = require('../utils/client');
+
+const newClient = new client.Client(keys);
+
+newClient.connectToServer('http://pathways.mitre.org:8180/fhir');
+newClient.connectToServer('http://pathways.mitre.org:8190/fhir');
```

Comment:
These two lines are just for testing right? They should be removed before merging